pkg/aws/ocpgwdeployer.go

```diff
@@ -83,14 +86,18 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 
 	status.Success(messageValidatedPrerequisites)
 
-	status.Start("Creating Submariner gateway security group")
+	gatewaySG := ""
```

Comment:
This can be declared local on L94.

```suggestion
```

Reply:
gatewaySG also used in L141 outside of if scope.

Reply:
I missed that but should L141 even execute if `gatewaySG` is not set?

Reply:
>I missed that but should L141 even execute if gatewaySG is not set?
Yep, it should. deployGateway should be called regardless of LB configuration and sg.